doc/release-notes/release-6.sgml

```diff
@@ -130,10 +141,16 @@ This section gives an account of those changes in three categories:
 	<p>This feature has been of limited use since AsyncCalls feature
 	   took over much of the CBDATA functionality.
 
+	<tag>--enable-gnuregex</tag>
+	<p>The GnuRegex library has been removed. C++17 standard library is used instead.
```

Comment:
If we do not use STL for some regular expressions that were using GnuRegex in old `--enable-gnuregex` builds, then we should remove the second claim:

```suggestion
	<p>The GnuRegex library has been removed.
```

I would also polish the wording further:

```suggestion
	<p>Squid no longer ships with a built-in GnuRegex implementation.
```

... or something like that. Most folks probably do not view Squid distribution as something that contains a GnuRegex _library_ and we probably still support Squid builds that use an external GnuRegex library (indirectly).